src/scales/categoryScale.ts

```diff
@@ -3,7 +3,6 @@
 module Plottable {
 export module Scales {
   export class Category extends Scale<string, number> {
-    protected _d3Scale: D3.Scale.OrdinalScale;
```

Comment:
Can we go around this? I really don't like the casts. Do something like
`this._d3Scale = <cast_here>super._d3Scale`
This will solve that issue